openshift_tooling/openshift_labeler/openshift_label.yml

```diff
@@ -0,0 +1,70 @@
+---
+- hosts: masters[0]
+  user: root
+  vars:
+    - file: /run/openshift/register.hosts
+    - tooling_inv: /root/tooling_inventory
+    - register_all_nodes : False
+    - oc_client_url: "https://github.com/openshift/origin/releases/download/v1.2.1/openshift-origin-client-tools-v1.2.1-5e723f6-linux-64bit.tar.gz"
+  tasks:
+    - name: create openshift directory
+      file: 
+        path: /run/openshift 
+        state: directory
+
+    - name: clear file contents
+      file: 
+        path: "{{ file }}" 
+        state: absent
+
+    - name: copy, set permissions on the register script
+      copy:
+        src: generate.sh
+        dest: /run/openshift/generate.sh
+        owner: root
+        group: root
+        mode: 0755
+
+    - name: add masters to the file
+      shell: echo -e {{ item }} master  >> {{ file }}
```

Comment:
Why not `lineinfile`?

Reply:
Yes, we can use lineinfile. Will update the PR.